guides/hosting/configurations/shopware/staging.md

```diff
@@ -0,0 +1,186 @@
+---
+nav:
+  title: Staging
+  position: 10
+
+---
+
+# Staging
+
+Since Shopware 6.6.1.0, Shopware has a integrated Staging Mode. This mode prepares the Shop to be used in a staging environment. This means that the shop is prepared to be used in a test environment, where changes can be made without affecting the live shop.
```

Comment:
```suggestion
Since Shopware 6.6.1.0, Shopware has an integrated Staging Mode. This mode prepares the Shop to be used in a staging environment. This means that the shop is prepared to be used in a test environment, where changes can be made without affecting the live shop.
```